source/world_builder/features/interface.cc

```diff
@@ -89,8 +88,8 @@ namespace WorldBuilder
 
                 prm.declare_entry("interpolation",Types::String("global"),
                                   "What type of interpolation should be used to enforce the minimum points per "
-                                  "distance parameter. Options are global, none, linear, monotone spline and "
-                                  "continuous monotone spline interpolation. If this "
+                                  "distance parameter. Options are 'global' and "
+                                  "'continuous monotone spline' interpolation. If this "
```

Comment:
Could you add a note here that this parameter is deprecated?

Reply:
is this because the global one is also only "continuous monotone spline"?

Reply:
yes, and I probably want to get rid of this option before the 1.0 release.